assets/tests/components/zoomLevelWrapper.test.tsx

```diff
@@ -0,0 +1,49 @@
+import "@testing-library/jest-dom"
+import { render, screen } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import React from "react"
+import { MapContainer } from "react-leaflet"
+import ZoomLevelWrapper from "../../src/components/ZoomLevelWrapper"
+
+const originalScrollTo = global.scrollTo
+// Clicking/moving map calls scrollTo under the hood
+jest.spyOn(global, "scrollTo").mockImplementation(jest.fn())
+
+afterAll(() => {
+  global.scrollTo = originalScrollTo
+})
+
+const renderProps = (zoomLevel: number) => <p>{zoomLevel}</p>
+
+const getMapZoomInButton = () => {
+  return screen.getByRole("button", { name: "Zoom in" })
+}
```

Comment:
praise+thought(non-blocking): I'm glad you pulled this out into a funciton!
I was really liking writing my tests with functions to get the elements we work with or expect to exist in a certain form. 

Since we now have this specific function twice, do y'all want to consolidate common elements we use into a helper file somewhere in tests? 
I couldn't figure out a good place to put them for sharing.

Reply:
@mbta/skate-developers

Reply:
I definitely lifted this from your earlier tests - it makes writing/reading tests much easier! Agree it would be nice to consolidate them, though I'm not sure at what level. Seems like just one file could cause naming collisions and confusion, but I'm not sure what the appropriate line to draw would be

Reply:
I originally was thinking that maybe keeping the relevant functions with their tests would be a good idea, but I think that means it would evaluate all the root level function calls (mock, describe, test) when trying to import a single function. So maybe it'd be better to have a file tree of matching "get component definitions" grouped by which `asset/components/*.tsx` it's from when it becomes relevant for each component we write tests for?

Reply:
Yeah I like that idea! I'll plan on adding that in the [following PR](https://github.com/mbta/skate/pull/1906) which introduces more query functions.

Reply:
Oh that's cool! I was wondering how we might make it more mailable/configurable! I'd say add it heh.

It seems like it hasn't been updated in a year or two, which is likely fine, but we should probably be aware that we might need to do security updates manually if it's unable to be maintained upstream.

I still wish there was a way to add extra search props, like `checked` `selected` `hidden`, but it's probably fine without it.